examples/config/roger-mesos-toolsconfig.yml

```diff
@@ -0,0 +1,26 @@
+default_environment: dev
```

Comment:
For the endpoints, and registry - Remove all the `moz.com` urls and replace with `example.com` urls and no port numbers. e.g. `chronos.example.com`, `marathon.example.com`, `registry.example.com` and so on...